BookPlayer/Library/ItemListViewController.swift

```diff
@@ -380,6 +400,27 @@ extension ItemListViewController {
             self.tableView.contentInset.bottom = rootViewController.miniPlayerIsHidden ? 0.0 : 88.0
         }
     }
+
+    @objc func selectButtonPressed(_ sender: Any) {
+        if self.tableView.numberOfRows(inSection: 0) == (self.tableView.indexPathsForSelectedRows?.count ?? 0) {
+            for row in 0..<self.tableView.numberOfRows(inSection: 0) {
+                self.tableView.deselectRow(at: IndexPath(item: row, section: 0), animated: true)
```

Comment:
Let's also replace this initialization of `IndexPath(item: row, section: 0)` with our own initializer `IndexPath(row: row, section: .data)`

Reply:
there's a second occurrence of this in this same function